src/main/proto/cinemaHall.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+
```

Comment:
we have a convention for proto-naming, it should be in snake_case. so, this file should be named cinema_hall.proto.  Also, parameters should be named in snake_case (showTime  -> show_time)

Reply:
Done!